corehq/apps/reports/management/commands/project_stats_report.py

```diff
@@ -43,6 +43,47 @@
     SimpleTableWriter,
     TableRowFormatter,
 )
+from casexml.apps.phone.models import SyncLogSQL
+
+
+RESOURCE_MODEL_STATS = [
+    'total_users',
+    'monthly_forms_per_user',
+    'monthly_cases_per_user',
+    'monthly_user_form_stats_expanded',
+    'monthly_user_case_stats_expanded',
+    'monthly_user_cases_updated',
+    'case_index_ratio',
+    'attachments',
+    'forms_total',
+    'cases_total',
+    'case_transactions_per_form',
+    'case_transactions_total',
+    'case_indices',
+    'synclogs_per_user',
+    'static_datasources',
+    'dynamic_datasources',
+    'datasources_info',
+    'devicelogs_per_user',
+]
+
+
+class ResourceModel(object):
```

Comment:
Why not use a dataclass or attrs class?

Reply:
bump

Reply:
> Why not use a dataclass or attrs class?

No particular reason other than the fact that I didn't think of it. However, after having experimented with it I do agree it's an attractive solution.